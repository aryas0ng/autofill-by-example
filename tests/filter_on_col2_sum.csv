0,1,4
4,1,4
3,4,3
8,6,23
10,6,23
5,6,
4,19,
4,19,
21,19,
21,19,
10,9,
7,7,
4,7,
3,3,
3,3,
16,3,
16,1,
27,1,
8,1,
3,3,
3,3,
11,8,
4,8,