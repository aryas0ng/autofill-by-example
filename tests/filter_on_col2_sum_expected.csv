0,1,4
4,1,4
3,4,3
8,6,23
10,6,23
5,6,23
4,19,50
4,19,50
21,19,50
21,19,50
10,9,10
7,7,11
4,7,11
3,3,22
3,3,22
16,3,22
16,1,51
27,1,51
8,1,51
3,3,6
3,3,6
11,8,15
4,8,15